common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -328,7 +330,8 @@ class @Problem
   reset_internal: =>
     Logger.log 'problem_reset', @answers
     $.postWithPrefix "#{@url}/problem_reset", id: @id, (response) =>
-        @render(response.html)
+        reset_error = response['error']
+        @render(response.html, reset_error)
```

Comment:
@mehrozezahid Please rename the all the `reset_error` variables to `error` or `error_message` to make this generic.